modules/apps/commerce/commerce-wish-list-web/src/main/resources/META-INF/resources/wish_list_item_action.jsp

```diff
@@ -22,6 +22,7 @@ CProduct cProduct = commerceWishListItem.getCProduct();
 	<c:when test="<%= cpInstance != null %>">
 		<commerce-ui:add-to-cart
 			alignment="left"
+			CPCatalogEntry="<%= commerceWishListDisplayContext.getCPCatalogEntry(cProduct.getPublishedCPDefinitionId()) %>"
```

Comment:
looking at the code this is the right fix, but why cpCatalogEntry is not mandatory if without this nothing makes sense?
I think we should make it mandatory

Reply:
Yes, let' open a story for this

Reply:
https://liferay.atlassian.net/browse/COMMERCE-12737